src/components/pages/EditorModals.jsx

```diff
@@ -59,6 +59,27 @@ const EditorModals = ({
     onClose()
   }
 
+  const onExternalMediaSave = (data) => {
+    const { selectedMediaPath, altText } = data
+    const cm = simpleMde.codemirror
+
+    cm.setCursor(linePos)
+    cm.setSelection(linePos)
+
+    if (mediaType === "files")
+      cm.replaceSelection(
+        `[${altText}](${selectedMediaPath.replaceAll(" ", "%20")})`
+      )
+    if (mediaType === "images") {
+      cm.replaceSelection(
+        `![${altText}](${selectedMediaPath.replaceAll(" ", "%20")})`
+      )
+    }
+    // set state so that rerender is triggered and image is shown
+    onSave(simpleMde.codemirror.getValue())
+    onClose()
+  }
```

Comment:
nit: this doesn't appear to be any different from `onMediaSave` defined above?

Reply:
Yeah you're right, removed in 516be476!